cloudigrade/util/aws/cloudtrail.py

```diff
@@ -0,0 +1,130 @@
+"""Helper utility module to wrap up common AWS CloudTrail operations."""
+import logging
+
+from botocore.exceptions import ClientError
+from django.conf import settings
+from django.utils.translation import gettext as _
+
+
+logger = logging.getLogger(__name__)
+
+
+def configure_cloudtrail(session, aws_account_id,
+                         include_global_service_events=True,
```

Comment:
It doesn't look like these two parameters are exposed to the end user.  Do we need parameters now?

Reply:
No we do not. I'll remove them.